dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -567,6 +559,9 @@ def generate_decoders_data(data_dir, data_extension):
     # File reader won't work, so I need to load audio files into external_source manually
     fnames = test_utils.filter_files(data_dir, data_extension)
 
+    nfiles = len(fnames)
+    for i in range(len(fnames), 10): # At leat 10 elements
+        fnames.append(fnames[-1])
```

Comment:
Could you add `TODO: remove` here? I'll remove these 3 lines with my `more_audio_data` PR

Reply:
OK, I'll do